db/health/schema.rb

```diff
@@ -861,6 +863,7 @@
     t.integer "rejected_reason", default: 0, null: false
     t.integer "patient_id"
     t.integer "accountable_care_organization_id"
+    t.datetime "effective_date", default: -> { "now()" }
```

Comment:
This looks suspicious.

Reply:
Agreed! I merged pre-release and ran `bin/update` to make sure the schemas were up to date...